runtime/actions/parsing.go

```diff
@@ -183,29 +183,21 @@ var toDate = func(value any) (types.Date, error) {
 //
 // e.g. for InlineFile inputs, whcih are given as a dataURL string, they need to be transformed into an object
 // including the typename
-func TransformCustomFunctionsInputTypes(schema *proto.Schema, action *proto.Action, input any) (any, error) {
-	inputsAsMap, isMap := input.(map[string]any)
-	if !isMap {
-		return input, nil
-	}
-
-	msg := proto.FindMessage(schema.Messages, action.InputMessageName)
-	if msg == nil {
-		return input, nil
-	}
+func TransformCustomFunctionsInputTypes(schema *proto.Schema, messageName string, input map[string]any) (map[string]any, error) {
+	message := proto.FindMessage(schema.Messages, messageName)
```

Comment:
```suggestion
	message := proto.FindMessage(schema.Messages, messageName)
	if message == nil {
		return input, nil
	}
```

In the case of a custom action with `Any`, the message won't exist and the following bit of code will throw a panic (`message.HasFiles()`)

Reply:
Nice